source/aws-ruby-sdk/faraday_helper.rb

```diff
@@ -0,0 +1,22 @@
+require 'faraday'
+
+module IdentityIdpFunctions
+  module FaradayHelper
```

Comment:
I did this as a mixin, but open to making it something we instantiate? No strong feelings here